Source/Core/Core/IOS/Network/KD/VFF/VFFUtil.cpp

```diff
@@ -245,17 +253,16 @@ ErrorCode OpenVFF(const std::string& path, const std::string& filename,
   VffFatFsCallbacks callbacks;
   ErrorCode return_value;
   Common::RunInFatFsContext(callbacks, [&]() {
-    auto file = fs->OpenFile(PID_KD, PID_KD, path, FS::Mode::ReadWrite);
-    if (!file)
+    auto temp = fs->OpenFile(PID_KD, PID_KD, path, FS::Mode::ReadWrite);
+    if (!temp)
     {
       ERROR_LOG_FMT(IOS_WC24, "Failed to open VFF at: {}", path);
       return_value = WC24_ERR_NOT_FOUND;
       return;
     }
 
-    s_vff = file.operator->();
+    callbacks.m_vff = reinterpret_cast<FS::FileHandle*>(&temp);
```

Comment:
This looks... sketchy to me, I don't think that cast actually does the right thing. `callbacks.m_vff = &*temp;` is probably the best you can do here.